selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -411,27 +478,52 @@ public void waitTextIntoMacrosContainer(final String expectedText) {
                 webDriver -> getAllVisibleTextFromMacrosContainer().contains(expectedText));
   }
 
+  /**
+   * Gets all visible text from proposal area of the "Command Macros" form.
+   *
+   * @return visible text from proposal area
+   */
   public String getAllVisibleTextFromMacrosContainer() {
     waitCommandsMacrosIsOpen();
     return macrosContainer.getText();
   }
 
+  /**
+   * Performs entering the specified {@code item} by single clicking and sending of the "Enter" key.
+   *
+   * @param item macro body in format "${current.project.path}" without description
+   */
   public void enterMacroCommandByEnter(String item) {
     selectMacroCommand(item);
     seleniumWebDriverHelper.sendKeys(ENTER.toString());
   }
 
+  /**
+   * Performs entering the specified {@code item} by single clicking and performing double clicking.
+   *
+   * @param item macro body in format "${current.project.path}" without description
```

Comment:
It sounds over complicated as for me. how about simpler one: `Enters macro command {@code item} by pressing Enter key`